src/common/i18n/locales/en-US.json

```diff
@@ -1794,7 +1794,59 @@
+      "what-information-feed-body": "<p>Feed includes brief information about Post author, Post title, which Community it was published in, post rewards, how many upvotes and comments it get. To perform additional activity with the post, just click on the three dots under the post.</p>",
+      "what-new-feed-header": "What does the New feed show?",
+      "what-new-feed-body": "<p> The newest posts sorted by the published time are shown in the New feed. </p>",
+      "how-change-design-header": "How to change feed design?",
+      "how-change-design-body": "<p>Click on the grid icon at the top right of the feed. Choose one of the look designs: Classic or Grid. </p>",
+      "what-is-deck-header": "What is Deck wiew on Ecency?",
+      "what-is-deck-body": "<p>From other Classic and Grid wiews there is another wiew called Deck. Decks makes easy to manage your account. With decks not only you get to see multiple things in parallel but you can configure each decks and shuffle things around.</p>",
+      "what-deck-types-header": "What-are-the deck-types",
+      "what-deck-types-body": "By clicking + icon you will have new pop-up window that is shown deck tpyes. You can choose any type of deck and each deck has its own setup. For example choosing User deck you can see posts created user you want to see.",
+      "what-rising-feed-header": "What does Rising feed show?",
+      "what-rising-feed-body": "<p>Risind feed shows posts that have more comments and votes compare to others on entire platform.</p>",
+      "how-sort-rising-header": "How to sort Rising feed?",
+      "how-sort-rising-body": "The rising feed can be sorted by today, this week, this month, this year, and all time posts created during the specular time. Choose one of them and get a chance of seeing posts that become rising you wanted time.",
+      "what-controversial-feed-header": "What does controversial feed show?",
+      "what-controversial-feed-body": "<p>Controversial feed shows posts that have a relatively equal number of upvotes and downvotes. Sometimes posts get a lot of upvotes with little rewards.</p>",
+      "how-sort-controversial-header": "How to sort Controversial feed?",
+      "how-sort-controversial-body": "<p>Controversial feed can be sorted by today, this week, this month, this year and all the time posts that is created during the specular time.</p>",
+      "how-many-witnesses-header": " How many witnesses can I vote?",
+      "how-many-witnesses-body": "<p>Each account can vote for up to 30 witnesses.</p>",
+      "sort-witnesses-page-header": "How to sort witnesses page?",
+      "sort-witnesses-page-body": "<p>At the end of the Witnesses page, click on the Sort button to display options for sorting witnesses. You can sort them by rank, name and fee. You can choose any option as your wish for example choosing a rank option will give you witnesses according to the order of voting rank.</p>",
+      "how-vote-proposals-header": "How to vote proposals?",
```

Comment:
We can add also questions like: 
how to search witness? (using search input)
how to see which witnesses account votes on? (by adding `?voter=username` on url)
same questions for proposals...